apps/pie-storybook/stories/pie-textarea.stories.ts

```diff
@@ -234,15 +225,24 @@ const ExampleFormTemplate: TemplateFunction<TextareaProps> = () => html`
     </form>
 `;
 
+const WithLabelTemplate: TemplateFunction<TextareaProps> = (props: TextareaProps) => html`
+        <p>Please note, the label is a separate component. See <pie-link href="/?path=/story/form-label">pie-form-label</pie-link>.</p>
+        <pie-form-label for="${ifDefined(props.name)}" trailing="${props.maxLength ? `${props.value.length}/${props.maxLength}` : nothing}">Label</pie-form-label>
```

Comment:
> I don't know if passing the maxLength calculations on to the consumer is the right thing to do. It feels like something we should be able to provide out of the box.

@jamieomaguire, just in case it wasn't very clear. the actual validation will be done using the max length attribute by browsers. users just need to construct the text and I don't think that's complex considering why we are doing the spike